### ISSUE TITLE
Update operation form with latest information from the business area

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#200]
#### Description:

The business area has updated the requirements spreadsheet (but we're still waiting on a list of Reporting Activities and Regulated Product Name(s)).

Wireframe(s) - https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=1307%3A449&mode=design&t=fSfIwkTtCNyd48DC-1

#### Acceptance Criteria:

Given I an app user
When I navigate to any of the forms
Then I only see the latest fields (see dev checklist for more details)

Given I am a developer
When I use the db
Then there are no unused fields (the abstract base classes can stay)

#### Development Checklist:

- [x] Remove any fields that are marked NA from both db and forms
- [x] Add any missing tier 1 fields
- [x] In the operation type dropdown, give users two options, Single Facility Operation or Linear Facilities Operation
- [x] Remove the Operation type section (all of step 2)
- [x] Renumber the remaining sections
- [x] Remove any tier 2 or 3 fields that say "remove"
- [x] Any tier 2 or 3 fields that say "stay" should stay in the form but be optional
- [x] Users need to be able to select multiple Reporting Activities and Regulated Product Name(s), so update db and form (see design)
- [x] Update playwright `operation.spec.ts` (just need to change which fields are filled out)
- [x] Write jest tests
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [x] Acceptance criteria are tested by the CI pipeline
- [x] UI meets accessibility requirements
- [x] Configuration changes are documented, documentation and designs are updated
- [x] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [x] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [x] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by BCerki** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Team-Moose-Sprint-Board-60d216175a2689000ec774d0/issues/bcgov/cas-registration/200?planning-poker) with Zenhub @BCerki @dleard @joshgamache @JoshLarouche @mikevespi @Sepehr-Sobhani @shon-button
